### PR TITLE
[WRITING CODE] Updating and simplifying ethics 

### PR DESCRIPTION
**Pull request by ehumph**
Simplifying the ethics box. Much of the ethics background has been taken out. Also shortened the language of the introduction.

### CODE REVIEW

File: 02-writing_code.Rmd [hunk 1]
```diff
@@ -4,77 +4,49 @@
...
-# Writing and Debugging Code
+# Writing Code
 
 In today's digitally-driven world, coding has become an essential skill. It is used to create websites, mobile applications, and sophisticated software systems. Contrary to popular belief, coding is not only for those who possess an innate talent for programming. With the right resources and a willingness to learn, anyone can become proficient in coding.
 
 Practice is crucial to mastering the art of coding, and those who are willing to commit themselves to the learning process can achieve great results. Writing code can also be a rewarding and fulfilling hobby, too. Creating your own software, building a website, or automating a task can be immensely satisfying, and can provide a sense of accomplishment and pride.
 
 Moreover, writing code can be a form of creative expression. Just as musicians use notes to compose music and artists use paint to create visual art, programmers use code to create digital experiences. From building websites to creating video games, code can be used to bring ideas to life in exciting and innovative ways.
 
+[FIGURE SHOWING CODING AS A CREATIVE ART???]
```
Comment (avahoffman) [inline 02-writing_code.Rmd:15]: ooo I have some ideas!

Comment (ehumph) [reply 02-writing_code.Rmd:15]: Ooo, yay!

Comment (avahoffman) [reply 02-writing_code.Rmd:15]: lol https://docs.google.com/presentation/d/1MCNeSO4aOm1iESWDLOGTcx3aLEbnu8UttV0QGVAeafE/edit#slide=id.g230c4a00725_1_0

Comment (ehumph) [reply 02-writing_code.Rmd]: It is so much better than my wildest dreams. ADORABLE DINOS FOR THE WIN

**Comment by ehumph** [comment]
I think I will switch to the ai answer box format for at least the function writing example, but that can be a separate PR.

File: 02-writing_code.Rmd [hunk 2]
```diff
@@ -4,57 +4,51 @@
...
 
 Moreover, writing code can be a form of creative expression. Just as musicians use notes to compose music and artists use paint to create visual art, programmers use code to create digital experiences. From building websites to creating video games, code can be used to bring ideas to life in exciting and innovative ways.
 
-Ultimately, the ability to write code is a valuable asset in today's world, with increasing demand for skilled programmers in a wide range of industries. Whether as a career or hobby, coding can be both personally and professionally rewarding. The accessibility and versatility of coding make it an excellent skill for anyone to learn, regardless of their background or previous experience. 
-
-
-## Using AI to write code
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "The dinosaur cartoon is wearing a beret and holding a paintbrush that is is using to paint code onto an easel."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MCNeSO4aOm1iESWDLOGTcx3aLEbnu8UttV0QGVAeafE/edit#slide=id.g230c4a00725_1_0")
```
Comment (cansavvy) [inline 02-writing_code.Rmd:16]: ❤️

File: 02-writing_code.Rmd [hunk 3]
```diff
@@ -74,65 +68,236 @@ The second rule about using AI is to **test everything**. You are responsible fo
...
+:::
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Phind suggests code modifications that allow you to reverse strings that appear in the third column."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MCNeSO4aOm1iESWDLOGTcx3aLEbnu8UttV0QGVAeafE/edit#slide=id.g230c4a00725_31_84")
+```
+
+
+
+
+## Example Two: Debugging Code
```
Comment (cansavvy) [inline 02-writing_code.Rmd:127]: I assume these are forthcoming in a future PR?

File: 02-writing_code.Rmd [hunk 4]
```diff
@@ -74,65 +68,236 @@ The second rule about using AI is to **test everything**. You are responsible fo
...
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MCNeSO4aOm1iESWDLOGTcx3aLEbnu8UttV0QGVAeafE/edit#slide=id.g230c4a00725_31_34")
+```
+
+
+
+
+After you find a solution that works for you, you should always make a habit of asking the AI where it got the code. Never, ever, ever run code without knowing where it came from.
+
+:::{.query}
+"Did you use any libraries or repos in this code?"
```
Comment (cansavvy) [inline 02-writing_code.Rmd:171]: What happens if we ask it more generally what its sources for the code are?

**Review by cansavvy — APPROVED** [review]
I think this looks great! I like the streamlined-ness of it compared to before. 

It looks like there are still examples you are planning to add? Are you planning on doing that in this PR or future PRs?

File: 02-writing_code.Rmd (hunk 3 above)
Comment (ehumph) [reply 02-writing_code.Rmd:127]: Yes, I have vague ideas for them, but panicked yesterday that this PR was getting too long.

**Comment by avahoffman** [comment]
@ehumph Is this outdated now? Should we delete this branch?

**Comment by ehumph** [comment]
@avahoffman yes

**Pull request closed without merging**